tests/testImageMapReduce.py

```diff
@@ -130,58 +142,122 @@ class ImageMapReduceTest(lsst.utils.tests.TestCase):
     """A test case for the image gridded processing task
     """
     def setUp(self):
+        self._makeImage()
```

Comment:
Since you're using `msg=` now, probably set `self.longMessage=True`, to get both your custom messages, and the default unittest ones.